k8s/rabbitmq/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,4 +1,29 @@
 actions:
+- name: RabbitMQ test
+  bashTest:
+    script: |-
+      SERVICE="${APP_INSTANCE_NAME}-rabbitmq-svc.${NAMESPACE}.svc.cluster.local"
+
+      USERNAME=$(kubectl get sts -oyaml \
+        --namespace "${NAMESPACE}" "${APP_INSTANCE_NAME}-rabbitmq" | \
+        grep -A1 name:.RABBITMQ_DEFAULT_USER | sed -n '/value: /s/.* //p')
+
+      PASSWORD=$(kubectl get secret "${APP_INSTANCE_NAME}-rabbitmq-secret" \
+        --namespace "${NAMESPACE}" \
+        --output=jsonpath='{.data.rabbitmq-pass}' | base64 -d)
+
+      wget -t3 "http://${SERVICE}:15672/cli/rabbitmqadmin"
+      chmod +x rabbitmqadmin
+
+      MESSAGE='simple message'
+      ./rabbitmqadmin -u"${USERNAME}" -p"${PASSWORD}" -H"${SERVICE}" declare queue name=testq durable=false 2>&1
+      echo -n "${MESSAGE}" | ./rabbitmqadmin -u"${USERNAME}" -p"${PASSWORD}" -H"${SERVICE}" publish routing_key=testq
+      if ! ./rabbitmqadmin -u"${USERNAME}" -p"${PASSWORD}" -H"${SERVICE}" get queue=testq | grep -q "${MESSAGE}"; then
+        exit 1
```

Comment:
Do we actually need to check the result of that call inside if, once we don't have any further commands and the result is verified by the exitCode either way?